<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qobjectbrowser</class>
 <widget class="QWidget" name="qobjectbrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QObject Browser</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <layout class="QVBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>&lt;b&gt;Class:&lt;/b&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>&lt;b&gt;Base Class:&lt;/b&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="class_name">
           <property name="text">
            <string>[Class Name]</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="base_class_name">
           <property name="text">
            <string>[Base Class Name]</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="parent_button">
           <property name="text">
            <string>Up to parent...</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>&lt;b&gt;Parent:&lt;/b&gt;</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="parent_class_name">
             <property name="text">
              <string>[Parent Class Name]</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Children</string>
        </attribute>
        <layout class="QGridLayout">
         <property name="margin">
          <number>9</number>
         </property>
         <property name="spacing">
          <number>6</number>
         </property>
         <item row="1" column="1">
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="update_children_button">
           <property name="text">
            <string>Update Children</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QTableWidget" name="children_table"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="properties_tab">
        <attribute name="title">
         <string>Properties</string>
        </attribute>
        <layout class="QGridLayout">
         <property name="margin">
          <number>9</number>
         </property>
         <property name="spacing">
          <number>6</number>
         </property>
         <item row="1" column="1">
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QTableWidget" name="properties_table"/>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="update_properties_button">
           <property name="text">
            <string>Update Properties</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="signals_tab">
        <attribute name="title">
         <string>Signals</string>
        </attribute>
        <layout class="QVBoxLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="margin">
          <number>9</number>
         </property>
         <item>
          <widget class="QTableWidget" name="signals_table"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="slots_tab">
        <attribute name="title">
         <string>Slots</string>
        </attribute>
        <layout class="QVBoxLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="margin">
          <number>9</number>
         </property>
         <item>
          <widget class="QTableWidget" name="slots_table"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="methods_tab">
        <attribute name="title">
         <string>Methods</string>
        </attribute>
        <layout class="QVBoxLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="margin">
          <number>9</number>
         </property>
         <item>
          <widget class="QTableWidget" name="methods_table"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
